metrics/utils/cfs.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
```

Comment:
Directory structure nit -- this utility in not unique to metrics. I'd much prefer to see it in a top level ``utils`` directory.

Reply:
I'm fine with that :)